src/backend/functions/users.ts

```diff
@@ -89,6 +92,8 @@ const logUserIn: ApiRouteFunction = async (_params, event) => {
     user = createdUser;
   }
 
+  const accessToken = jwt.sign(user, process.env.JWT_SECRET as string, { expiresIn: '20mins', algorithm: 'HS256' });
```

Comment:
How did you decide on the algorithm and the expiration? What are the options?

Reply:
The options are HS256 and RS256. HS256 uses a shared secret while RS256 uses a public and a private key

Reply:
Awesome!